lib/tech_docs_html_renderer.rb

```diff
@@ -0,0 +1,7 @@
+require 'middleman-core/renderers/redcarpet'
+
+class TechDocsHTMLRenderer < Middleman::Renderers::MiddlemanRedcarpetHTML
+  def image(*args)
+    %(<a href="#{args.first}" target="_blank">#{super}</a>)
```

Comment:
Is the `args.first` thing a common pattern? Personally I think it'd be more readable if it was e.g.

```ruby
def image(path, *args)
  %(<a href="#{path}" target="_blank">#{super}</a>)
end
```

But happy to be overruled if this is a 'thing'.